Source/WebCore/Modules/WebGPU/Implementation/WebGPUDeviceImpl.cpp

```diff
@@ -141,8 +141,8 @@ RefPtr<Texture> DeviceImpl::createTexture(const TextureDescriptor& descriptor)
         convertToBackingContext->convertToBacking(descriptor.format),
         descriptor.mipLevelCount,
         descriptor.sampleCount,
-        static_cast<uint32_t>(backingTextureFormats.size()),
-        backingTextureFormats.data(),
+        backingTextureFormats.size(),
+        backingTextureFormats.size() ? backingTextureFormats.data() : nullptr,
```

Comment:
oh this is more consistent since a Vector of size=0 with non-zero capacity would have a non-null `.data()`?

Reply:
Exactly. And because there was at least one call site where we were checking for a non-null base pointer, I figured it would be best to align on always using null in this case.

Reply:
I mean it might have a null `.data()`; it depends what you've done with the vector beforehand.